RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 * [*] [internal] A refactor in weekly roundup notification scheduler. [#19422]
 * [*] Stories: Fixed an issue where the keyboard would overlap with the publish dialog in landscape. [#19350]
 * [*] Fixed an issue where the navigation bar becomes invisible when swiping back to Login Prologue screen.  [#19461]
+* [*] Fixed an issue where the prompt to install Jetpack is not shown when logout WordPress.com account [#19403]
```

Comment:
How about: `Fixed an issue where the notification tab's prompt view is not matched with the login account`

I want to keep the note as short as possible because it already contains the issue number `[#19403]`. If someone is interested in the fix, they can open the issue easily.